src/api/app/models/staging_workflow.rb

```diff
@@ -1,4 +1,12 @@
 class StagingWorkflow < ApplicationRecord
   belongs_to :project, inverse_of: :staging
-  has_many :staging_projects, class_name: 'Project', inverse_of: :staging_workflow, dependent: :nullify
+  has_many :staging_projects, class_name: 'Project', inverse_of: :staging_workflow, dependent: :nullify, autosave: true
+
+  after_initialize :init_staging_projects
+
+  def init_staging_projects
+    ['A', 'B'].each do |letter|
```

Comment:
[nitpick] Move the array to a constant
```
DEFAULT_STAGING_PROJECTS = ['A', 'B'].freeze
```

Reply:
Are we going to reuse this?

Reply:
What is not readable in ['A', 'B'] ?? :smile:

Reply:
Not the array but I don't know what the array is supposed to mean... Giving things a name makes it more readable so that I know immediately these are the default staging projects...